R/expand.R

```diff
@@ -20,7 +20,7 @@ expand <- function(x, approx_range) UseMethod("expand")
 #' That is, 3 days for day approximation, 3 months for month approximation,
```

Comment:
For uncertain dates with ? annotation, are we just dropping the annotation or expanding in full? (line 9) I think they're currently expanded if it's incomplete (i.e. 2000? expands to full range of dates in the year).

Reply:
Currently, we are dropping the "?" sign of unreliable dates in expand(). However, you are right, dates that are incomplete with the "?" (i.e. 2000?) are expanded. I will try to update the documentation to make this clearer! Thank you Jael